.github/workflows/build.yaml

```diff
@@ -0,0 +1,57 @@
+name: Build
+on: [workflow_dispatch, pull_request]
+env:
+  TERM: xterm-256color
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Build - Ubuntu
+            runs-on: ubuntu-latest
+
+          - name: Build - Windows (win32)
+            runs-on: windows-latest
+            cmake-args: -A Win32
+
+          - name: Build - Windows (win64)
+            runs-on: windows-latest
+            cmake-args: -D CMAKE_GENERATOR_PLATFORM=x64
```

Comment:
I suppose `-D CMAKE_GENERATOR_PLATFORM=x64` is unneeded.
1. I do not see any `CMAKE_GENERATOR_PLATFORM` usage in CMakeLists.txt
2. `NOT CMAKE_GENERATOR_PLATFORM STREQUAL Win32` should be used instead of `CMAKE_GENERATOR_PLATFORM STREQUAL x64` if it would be used.

Reply:
Removed.